java/main/com/google/privacy/differentialprivacy/LaplaceNoise.java

```diff
@@ -149,17 +150,58 @@ public ConfidenceInterval computeConfidenceInterval(
       long lInfSensitivity,
       double epsilon,
       @Nullable Double delta,
-      double confidenceLevel) {
-    // TODO: Implement confidence interval computation.
-    throw new UnsupportedOperationException("Not implemented yet.");
+      double alpha) {
+    checkConfidenceParameters(l0Sensitivity, lInfSensitivity, epsilon, delta, alpha);
+    ConfidenceInterval confInt =
+        computeConfidenceInterval(
+            noisedX, Noise.getL1Sensitivity(l0Sensitivity, lInfSensitivity) / epsilon, alpha);
+    return ConfidenceInterval.create(
+        Math.round(confInt.lowerBound()), Math.round(confInt.upperBound()));
   }
 
   private void checkParameters(double l1Sensitivity, double epsilon, @Nullable Double delta) {
-    DpPreconditions.checkEpsilon(epsilon);
+    DpPreconditions.checkEpsilonVeryStrict(epsilon);
     DpPreconditions.checkNoiseDelta(delta, this);
     DpPreconditions.checkL1Sensitivity(l1Sensitivity);
   }
 
+  private void checkConfidenceParameters(
+      int l0Sensitivity,
+      double lInfSensitivity,
+      double epsilon,
+      @Nullable Double delta,
+      double alpha) {
+    DpPreconditions.checkEpsilonStrict(epsilon);
+    DpPreconditions.checkNoiseDelta(delta, this);
+    DpPreconditions.checkSensitivities(l0Sensitivity, lInfSensitivity);
+    DpPreconditions.checkAlpha(alpha);
+  }
+
+  /**
+   * Computes a confidence interval that contains the raw value {@code x} passed to {@link
+   * #addNoise(double, int, double, double, Double)} with a probability equal to {@code 1 - alpha}
+   * based on the specified {@code noisedX} and lambda.
+   */
+  private ConfidenceInterval computeConfidenceInterval(
+      double noisedX, double lambda, double alpha) {
+    // Finding a symmetrical confidence interval around a Laplace of (0, lambda)
```

Comment:
As I mentioned in the Gaussian noise implementation, there is no need for a comment here.